training_config/structured_prediction/semantic_dependencies.json

```diff
@@ -2,6 +2,10 @@
     "dataset_reader":{
         "type":"semantic_dependencies"
     },
+    "dataset_reader":{
```

Comment:
Doesn't this need to be `validation_dataset_reader`?

Reply:
Right, I fixed this!